server/server.js

```diff
@@ -11,3 +11,27 @@
 // iterating through that list, display the string you built up.
 
 // Remember that the response is a stream object that must be closed.
+var http = require('http');
+var trucks = require('./trucks');
+var enhancedDate = require('./enhancedDate');
+var dayName = enhancedDate.getDayName;
+var monthName = enhancedDate.getMonthName;
+
+var availableTrucks = trucks.filterByDay(dayName);
```

Comment:
The trucks module exports a function, not an object, so trucks is now a function that you call directly. If you had exported filterByDay as a method on an object, then this line would work.